modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -173,7 +165,7 @@ function inope_policy_review_page($node) {
     // Build a table of results from other users.
     $build['table'] = array(
       '#type' => 'markup',
-      '#markup' => inope_policy_review_table($node, $account),
+      '#markup' => views_embed_view('policies', 'block_1'),
```

Comment:
Is the render array overkill with views_embed_view?  What does the output look like either way?  If you can get away with only views embed view and the output isn't disgusting, that'd be cleaner I think.